Packs/NozomiNetworks/Integrations/NozomiNetworks/NozomiNetworks.yml

```diff
@@ -0,0 +1,253 @@
+category: Network Security
+commonfields:
+  id: Nozomi Networks
+  version: -1
+configuration:
+- additionalinfo: Here you have to insert the endpoint url of your Nozomi Networks
+    appliance
+  defaultvalue: https://nozomi_instance.com
+  display: Endpoint url
+  name: endpoint
+  required: true
+  type: 0
+- additionalinfo: A user having APIs access privileges.
+  display: Username
+  name: username
+  required: true
+  type: 0
+- additionalinfo: User’s password.
+  display: Password
+  hidden: false
+  name: password
+  required: true
+  type: 4
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  hidden: false
+  name: proxy
+  required: false
+  type: 8
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- additionalinfo: 'Here you can decide from how many time from the past to get incidents.
+    Default is 7 days. some examples of valid input are: 30 seconds, 1 minute, 4 hours,
+    2 days...'
+  defaultvalue: 7 days
+  display: Get incidents from last
+  hidden: false
+  name: fetchTime
+  required: false
+  type: 0
+- additionalinfo: Get alert having risk higher or equal the one selected.
+  defaultvalue: '1'
+  display: Get alerts from risk
+  hidden: false
+  name: riskFrom
+  options:
+  - '1'
+  - '2'
+  - '3'
+  - '4'
+  - '5'
+  - '6'
+  - '7'
+  - '8'
+  - '9'
+  - '10'
+  required: false
+  type: 15
+- additionalinfo: In Nozomi platform there are alerts as a single events and there
+    is also the concept of incident, that is an aggegation of alert, by default we
+    just send alerts to demisto, if you flag this checkbox we also send incidents.
+    When a fetch integration was configured without incidents, if enabled means that
+    the new n2os incidents will be sent to XSOAR, but not the old one.
+  display: Fetch also Nozomi incidents
+  hidden: false
+  name: fecthAlsoIncidents
+  required: false
+  type: 8
+description: |-
+  The Nozomi Networks Guardian platform is a hardware or virtual appliance that is used to monitor OT/IoT/IT networks. It combines asset discovery, network visualization, vulnerability assessment, risk monitoring and threat detection in a single solution.
+    This integration is used to gather alert and asset information from Nozomi.
+display: Nozomi Networks
+name: Nozomi Networks
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: You can add a filter to get exactly the assets you want. For example
+        'where ip match 10.0.1.10', 'where vendor ==  Selta Telematica S.p.a'
```

Comment:
Let's mention here that the filer needs to be taken from the Nozomi UI.

Reply:
?

Reply:
You can use the filter in the nozomi UI as well as in demisto, nozomi ui does not have already done filters, so it's strange IMHO to ask the user to take filter from there.
The user have to know how to filter elements in nozomi platform, so I add